# High resolution protein architecture of a eukaryotic genome

## Matthew J. Rossi, Prashant K. Kuntala, William K.M. Lai, Naomi Yamada, Nitika Badjatia, Chitvan Mittal, Guray Kuzu, Kylie Bocklund, Nina P. Farrell, Thomas R. Blanda, Joshua D. Mairose, Ann V. Basting, Katelyn S. Mistretta, David J. Rocco, Emily S. Perkinson, Gretta D. Kellogg, Shaun Mahony, B. Franklin Pugh

### Center for Eukaryotic Gene Regulation, Department of Biochemistry and Molecular Biology, The Pennsylvania State University, University Park, Pennsylvania 16802, USA

### Correspondence: bfp2@psu.edu

### www.yeastepigenome.org
### GEO ID : GSE147927

Proteins are architected along eukaryotic chromosomes to maintain chromosome integrity and express thousands of genes. There exists no holistic single-base resolution map of their structural organization that reflects on genome function. Here we use ChIP-exo to define this structure in the yeast Saccharomyces. We identified 21 meta-assemblages consisting of >400 proteins related to replication origins, centromeres, subtelomeric regions, transposons, and RNA polymerase (Pol) I, II, and III transcription units.  Most Pol II promoters lacked a regulatory region by design. These essentially constitutive promoters comprised a short nucleosome-free region adjacent to a +1 nucleosome, which together bound TFIID to formed a pre-initiation complex (PIC). Positioned insulators protected core promoters from upstream events. Only 1/5th of all promoters have inducible regulation, wherein combinations of 78 sequence-specific transcription factors bound upstream to create a nucleosome-depletable regulatory region. We describe their structural interactions with the genome and cognate cofactors, including nucleosome and transcription regulators RPD3-L, SAGA, NuA4, Tup1, Mediator, and SWI/SNF. Together their assemblages are linked to PIC assembly involving primarily TBP rather than TFIID to achieve a defined integrated network of regulated transcription. 
